### PR TITLE
=doc #23487 add missing scala snippet in testing doc

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
Resolves #23487

File: akka-docs/src/test/scala/docs/testkit/TestkitDocSpec.scala [hunk 1]
```diff
@@ -96,6 +98,31 @@ class TestkitDocSpec extends AkkaSpec with DefaultTimeout with ImplicitSender {
...
+    testActor.tell("hello", ActorRef.noSender)
+    testActor.tell("hello", ActorRef.noSender)
+    testActor.tell("world", ActorRef.noSender)
+    testActor.tell(42, ActorRef.noSender)
+    //#test-expect
+    val hello: String = expectMsg("hello")
+    val any: String = expectMsgAnyOf("hello", "world")
+    val all: immutable.Seq[String] = expectMsgAllOf("hello", "world")
+    val i: Int = expectMsgType[Int]
+    expectNoMsg()
```
Comment (patriknw) [inline akka-docs/src/test/scala/docs/testkit/TestkitDocSpec.scala]: add `200.millis`, since the default is typically not what one want

**Review by patriknw — APPROVED** [review]
LGTM, after small adjustment

**Review by johanandren — APPROVED** [review]

**Comment by ktoso** [comment]
Amended on ☎️  on 🚋  Should be good though, added the 200 millis

**Comment by ktoso** [comment]
Rebased, paradox passed locally... not sure what was wrong on jenkins here.

**Comment by johanandren** [comment]
Small `k` vs big `K` in file name, case insensitive FS vs case sensitive?

File: akka-docs/src/main/paradox/scala/testing.md [hunk 2]
```diff
@@ -63,6 +63,9 @@ actor—are stopped.
 The above mentioned @scala[`expectMsg`]@java[`expectMsgEquals`] is not the only method for formulating
 assertions concerning received messages, the full set is this:
 
+Scala
+:   @@snip [TestKitDocSpec.scala]($code$/scala/docs/testkit/TestKitDocSpec.scala) { #test-expect }
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/testing.md]: It was a case-case... Hope it'll work now. Fixing those on OSX is such a pain

File: akka-actor/src/main/scala/akka/io/InetAddressDnsResolver.scala [hunk 3]
```diff
@@ -49,3 +57,45 @@ class InetAddressDnsResolver(cache: SimpleDnsCache, config: Config) extends Acto
...
 }
+
+/**
+ * Mirrors obtaining the settings from Security properties as implemented
+ * in [[sun.net.InetAddressCachePolicy]] features (mostly constants).
+ *
+ * Shadow impl needed in order to not depend on `sun.net` packages, retaining
+ * compatibility with those settings.
+ */
+@InternalApi private object InetAddressCachePolicyShadow {
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/io/InetAddressDnsResolver.scala]: and this was intended to be included in this PR?

Comment (ktoso) [reply akka-actor/src/main/scala/akka/io/InetAddressDnsResolver.scala]: Oh, no; it wasn't; sorry 😞

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Uff, finally passed ;)

**Pull request merged**